src/Symfony/Component/Form/Command/FormDebugCommand.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Form\FormRegistryInterface;
+use Symfony\Component\Form\ResolvedFormTypeInterface;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A console command for retrieving information about form types.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class FormDebugCommand extends Command
+{
+    private $formRegistry;
+    private $optionsResolver;
+    private $parents = array();
+    private $extensions = array();
+    private $namespaces = array(
```

Comment:
I'd suggest:

```diff
diff --git a/src/Symfony/Component/Form/Command/FormDebugCommand.php b/src/Symfony/Component/Form/Command/FormDebugCommand.php
index 5e3411939a..cbb871369a 100644
--- a/src/Symfony/Component/Form/Command/FormDebugCommand.php
+++ b/src/Symfony/Component/Form/Command/FormDebugCommand.php
@@ -33,17 +33,14 @@ class FormDebugCommand extends Command
     private $optionsResolver;
     private $parents = array();
     private $extensions = array();
-    private $namespaces = array(
-        'Symfony\\Bridge\\Doctrine\\Form\\Type',
-        'Symfony\\Component\\Form\\Extension\\Core\\Type',
-    );
+    private $namespaces;
 
-    public function __construct(FormRegistryInterface $formRegistry, array $namespaces = array())
+    public function __construct(FormRegistryInterface $formRegistry, array $namespaces = array('Symfony\\Component\\Form\\Extension\\Core\\Type'))
     {
         parent::__construct();
 
         $this->formRegistry = $formRegistry;
-        $this->namespaces = array_unique(array_merge($this->namespaces, $namespaces));
+        $this->namespaces = $namespaces;
     }
 
     /**
```

Registering the core types are enough to me in case the command is instantiated without providing namespaces explicitly. About the bridge, it'll be registered by the pass and is not really "core", so let's remove it.